gnovm/cmd/gno/lint.go

```diff
@@ -82,7 +83,28 @@ func execLint(cfg *lintCfg, args []string, io commands.IO) error {
 			})
 		}
 
-		// TODO: add more checkers
+		filepath.Walk(pkgPath, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				fmt.Fprintf(io.Err(), "Error walking %q: %v\n", path, err)
+				hasError = true
+			} else if !info.IsDir() {
+				data, err := osm.ReadFile(path)
+				if err != nil {
+					hasError = true
+					fmt.Fprintf(io.Err(), "readfile %q: %v\n", path, err)
+				} else {
+					for _, line := range bytes.Split(data, []byte{'\n'}) {
+						if bytes.Contains(line, []byte("// nolint")) {
```

Comment:
It may be better to remove the space so it becomes `//nolint`.  It would make it the same as in the [golangci-lint tool](https://github.com/golangci/golangci-lint).